app/src/main/res/layout/activity_item_news.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
rename this file because it have bad name. Only activities layouts should start from "activity" prefix. Use Item_ prefix for viewHolders.

Reply:
Also you can use app: attributes for your template. for expl. app:text="Title" or app:textColor. You will see how your layout look like without starting app. Just try it)